src/os/im/action/importactionmanager.js

```diff
@@ -610,3 +610,17 @@ os.im.action.ImportActionManager.prototype.onAddActionEntry_ = function(event) {
     os.alertManager.sendAlert(msg, os.alert.AlertEventSeverity.ERROR);
   }
 };
+
+
+
+/**
+ * @param {os.source.Request} source
+ * @return {boolean}
+ */
+os.im.action.ImportActionManager.prototype.hasActiveActions = function(source) {
+  var featureActions = this.getActionEntries(source.getId());
+
+  const hasActiveActions = (action) => action.enabled === true;
+
+  return featureActions.some(hasActiveActions);
+};
```

Comment:
Couple things here:
1. This function should take just an ID as an argument, not the whole source. This class generally stores things by ID and all off the function signatures for getters work that way. Also, just better to make an argument to a function like this the simplest possible thing that it can be.

2. Rather than referencing `action.enabled`, we prefer to use the getter whenever possible (in this case, `action.isEnabled()`. The flag is actually defined with bracket notation on `FilterEntry`, so referencing it with a . is potentially a problem when compiled.

3. It's simpler to just run the function as the argument to `some` rather than assigning it to a variable, i.e. 
`featureActions.some((action) => action.isEnabled());`